docs/content/downloads/releasenotes/release-0.7.0.md

```diff
@@ -0,0 +1,55 @@
+* [Zipped update site](https://download.eclipse.org/elk/updates/releases/0.7.0/elk-0.7.0.zip) (for offline use)
+* [Maven central](https://repo.maven.apache.org/maven2/org/eclipse/elk/) (for building pure Java projects that use ELK)
+
+
+
+## Details
+
+This is a major release which comes with quite a number of changes. Some of those are breaking changes, either in the usual API-breaking sense or in the sense that default layouts might look different. Those [issues](https://github.com/eclipse/elk/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Release+0.7.0%22+label%3Abreaking) and [pull requests](https://github.com/eclipse/elk/pulls?q=is%3Apr+is%3Aclosed+milestone%3A%22Release+0.7.0%22+label%3Abreaking) are now labeled with "breaking" to make such changes easier to spot.
+
+Here's a list of the most noteworthy changes. Head over to GitHub for [the full list](https://github.com/eclipse/elk/milestone/12?closed=1).
+
+
+### New Features and Enhancements
+
+* [#362](https://github.com/eclipse/elk/issues/362), [#613](https://github.com/eclipse/elk/pull/613): Most importantly, we finally have a new logo which appears on all of our websites. Joy!
+* [#105](https://github.com/eclipse/elk/issues/105), [#598](https://github.com/eclipse/elk/pull/598): Configuring spacings properly is one of the most complex things to get right. We thus added [a utility class](https://github.com/eclipse/elk/blob/master/plugins/org.eclipse.elk.core/src/org/eclipse/elk/core/util/ElkSpacings.java) to help ease the pain a little.
+* [#533](https://github.com/eclipse/elk/issues/533), [#535](https://github.com/eclipse/elk/pull/535): _ELK Box_ and _ELK Rectangle Packing_ now support [content alignment](https://www.eclipse.org/elk/reference/options/org-eclipse-elk-contentalignment.html).
+* [#593](https://github.com/eclipse/elk/pull/593): _ELK Rectangle Packing_ now supports a configurable target width that the algorithm will try to achieve.
+* [#344](https://github.com/eclipse/elk/issues/344): ELK's JSON support nur supports a more relaxed JSON style.
+
+
+### Changes
+
+* [#500](https://github.com/eclipse/elk/issues/500): The Xtext version ELK uses was upgraded to 2.20.
+* [#626](https://github.com/eclipse/elk/issues/626), [#634](https://github.com/eclipse/elk/pull/634): We changed the way port label placement is configured. Previously, there was a choice between inside and outside port labels, with other details configured in other options. We have now made the option an `EnumSet` to move port label options out of the `SizeOptions` enumeration and the `nextToPortIfPossible` option.
+* [#646](https://github.com/eclipse/elk/issues/646), [#647](https://github.com/eclipse/elk/pull/647): _ELK Layered_'s layout options `layering.layerID` and `crossingMinimization.positionID` were renamed to `layering.layerId` and `crossingMinimization.positionId`, respectively. This also impacts the associated layout option constants.
```

Comment:
I think they haven't been part of a previous release, have they?

Reply:
Oh, good point. @soerendomroes you know best: @uruuru is correct, isn't he?

Reply:
They are included in elk 6.1(I checked), but I guess no one consumed them yet (despite the KLighD keith branch).

Reply:
OK, then I'll leave them here. Thanks for checking. 🙂